src/player.c

```diff
@@ -885,6 +900,7 @@ void player_realdeath(Player *plr) {
 	plr->bomb_fragments = 0;
 	plr->voltage *= 0.9;
 	plr->lives--;
+	stats_append_life(&plr->stats);
```

Comment:
Case in point: I was pretty confused about this code fragment, appending a "life" when a "death" has occurred, until I realized what the correct interpretation is.